docs/wmco-prerequisites.md

```diff
@@ -40,7 +40,7 @@ Note: OpenShiftSDN networking is the default network for OpenShift clusters. It
 | Hybrid OVNKubernetes      | Supported Windows Server version                                      |
 | -----------               | -----------                                                           |
 | Default VXLAN port        | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2019|
-| Custom VXLAN port         | Windows Server Semi-Annual Channel (SAC): Windows Server 20H2|
+| Custom VXLAN port         | <ul><li>Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2022</li><li>Windows Server Semi-Annual Channel (SAC): Windows Server 20H2</li></ul>|
```

Comment:
Please stick to the existing formating
```suggestion
| Custom VXLAN port         | Windows Server Long-Term Servicing Channel (LTSC): Windows Server 2022|
|                                           | Windows Server Semi-Annual Channel (SAC): Windows Server 20H2|

Reply:
Or something to this effect without introducing any HTML tags.

Reply:
Tried to make things look decent without any embedded tags. LMK if it's clear.

Reply:
Markdown all the way 👍🏽